homeassistant/components/homewizard_energy/translations/en.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This file will be automatically generated based on `strings.json`. During development you can use `python3 -m script.translations.develop` to re-generate this file.

Reply:
Thanks. If I understand it correctly, I don't have to add the `translations/*.json` files to this PR because they are generated by the strings.json file?

Reply:
Correct.

Reply:
Updated `strings.json` and removed `translations/*.json` https://github.com/home-assistant/core/pull/55812/commits/ca98ab80a2112a767e03b825612eabfbd1251d2c